definitions/private-law/json/AuthorisationCaseField/ServiceOfApplication/AuthorisationCaseField.json

```diff
@@ -0,0 +1,282 @@
+[
```

Comment:
gatekeeping can be done by either judge/la or courtadmin. we need to give access to this 3 types of roles. Confirm with BA as well.